.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,32 @@
+name: Benchmark
+on:
+  push:
+    branches:
+      - main
+env:
+  DEFAULT_GO_VERSION: 1.16
```

Comment:
Yeah. That's not really something formally supported by the GH Action used here. It would post one comment for each Go version, which would be quite heavy.
And for the gh-pages support, I don't think they could manage that.